themes/themes/nushell-dark.nu

```diff
@@ -80,4 +81,8 @@ export def main [] { return {
     shape_table: blue_bold
     shape_variable: purple
     shape_vardecl: purple
+
+    background: dark_gray
+    foreground: default
+    cursor: red
```

Comment:
cursor: red? seems like an odd choice. aren't most cursors are the same color as text?

Reply:
yeah to be honest i had no idea what color to use here :grimacing: 

**so do we go with `default`**? :yum: 

> btw, when i add the following to my `config.nu`, after the theme is set
> ```nu
> $env.config.color_config.cursor = red
> ```
> it does not change anything around the cursor in the REPL :thinking: